aws/resource_aws_ssm_document.go

```diff
@@ -39,6 +39,32 @@ func resourceAwsSsmDocument() *schema.Resource {
 				Required:     true,
 				ValidateFunc: validateAwsSSMName,
 			},
+			"attachments": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"key": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								ssm.AttachmentsSourceKeySourceUrl,
```

Comment:
Additional value is available:

```suggestion
								ssm.AttachmentsSourceKeyAttachmentReference,
								ssm.AttachmentsSourceKeySourceUrl,
```